django/db/backends/base/creation.py

```diff
@@ -337,21 +337,37 @@ def mark_expected_failures_and_skips(self):
         from unittest import expectedFailure, skip
 
         for test_name in self.connection.features.django_test_expected_failures:
+            # test_method_name might actually be a class.
+            # And either way, test_case_name might actually be a module.
```

Comment:
I wanted to refactor to avoid this going forward, but short of changing `import_string()` to start throwing when given a module, I don't think we have enough information to the tell the two cases apart.